src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -243,23 +305,29 @@ export default function SpeedUpAndCancelSheet() {
           if (type === SPEED_UP) {
             Alert.alert(
               'Unable to speed up transaction',
-              'There was a problem while fetching the transaction data. Please try again...'
+              'There was a problem while fetching the transaction data. Please try again...',
+              [
+                {
+                  onPress: () => goBack(),
+                },
+              ]
             );
-            goBack();
           }
           // We don't care about this for cancellations
         }
-        startPollingGasPrices();
-        // Always default to fast
-        updateGasPriceOption('fast');
       }
-    }, 300);
+    };
+
+    init();
 
     return () => {
```

Comment:
Nit: can we move the returned `stopPollingGasPrices` into the `useEffect` with `startPollingGasPrices` so we can keep these logically together in the code